src/scripts/statusbar.ts

```diff
@@ -3,12 +3,15 @@
  */
 namespace Scripts {
     export class Statusbar {
-        static create(targetSerial?: string) {
-            if (targetSerial === undefined) {
+        static create(o?: GameObject | string, coordinates?: ICoordinates) {
```

Comment:
ja by som bol rád, keby sme sa vyhýbali takýmto premenným typu `o`, `s` a podobne.. bolo by fajn to nazvať napr `mobile`, čo myslíte? potom človek keď sa v tom hrabe tak sa zle orientuje podľa mňa..

Reply:
muze bejt mobile.